health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreen/useCustomT.js

```
@@ -0,0 +1,14 @@
+
+export const useCustomT = (code) => {
+  const { locState, addMissingKey } = useAppLocalisationContext();
+  const currentLocale = Digit?.SessionStorage.get("initData")?.selectedLanguage || "en_IN";
+  const entry = locState?.find((item) => item.code === code);
+  if (!entry) {
+    addMissingKey(code); // Add the missing key
+    return ""; // Return the key as a placeholder
+  }
+  return entry[currentLocale] || ""; // Return the message or fallback to the key
+
+  return customT;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)